src/libopal.c

```diff
@@ -2312,6 +2312,13 @@ gen_asm_code (node_s *ast)
       gen_asm_code(ast->left);
       add_asm_code(ast->node_type, 0, NULL);
       break;
+    case nd_Ident:
```

Comment:
Get `location_offset` of identifier by calling `add_var` and add asm code `asm_Fetch` by calling `add_asm_code()`